### DISCUSSION TITLE
Editing header and footer of template pages

### DISCUSSION THREAD

**Discussion by chrismgill** [microweber/microweber#1064] [Q&A]
Hi. I am trying to build a site using a Softaculous install of Microweber. I don't seem to be able to edit the header or footer, eg to change the background.  Is  it  possible? How?

**Comment by ajitbhandari** [comment]
@chrismgill  to change the background you can use custom css or live css edit

**Comment by phausleitner** [comment]
Yes, Template Style Editor
![image](https://github.com/microweber/microweber/assets/108931257/a0848203-db57-474b-87fb-1c78f90d0896)
